### DISCUSSION TITLE
With Use Of API from the C#, how many links is going to generate per minute?

### DISCUSSION THREAD

**Discussion by IamAkashJoshi** [YOURLS/YOURLS#3015] [General]
I am using your application for a short URL. but today i observed that in one minute around 200 link is generated.

so can you tell is there any issue or i miss something?

**Comment by ozh** [comment]
I have no idea what you're talking about.

**Reply by IamAkashJoshi** [reply]
Ok let me explain.

I have 45000 URLs in one file. those URL need to do short.
so i write a for loop for the same and that for loop took to much time.

after i see the logs and realize that, YOURLS generates short URLs very slow. it's like 200 shorturl generated in one minute.

**Comment by dgw** [comment]
> so can you tell is there any issue or i miss something?

You did miss something: explaining the issue. I haven't a clue what you're asking. 200 API calls per minute could be great or terrible depending on where you have installed YOURLS (e.g. Raspberry Pi vs. Threadripper server).

**Reply by IamAkashJoshi** [reply]
Ok let me explain.

I have 45000 URLs in one file. those URL need to do short.
so i write a for loop for the same and that for loop took to much time.

after i see the logs and realize that, YOURLS generates short URLs very slow. it's like 200 shorturl generated in one minute.

**Reply by dgw** [reply]
That adds no new information. Again, on a Raspberry Pi, that might be amazing performance.

YOURLS (and your API client) will only run as fast as the machines running them can allow.

**Reply by IamAkashJoshi** [reply]
ok thank you
